src/linuxcan.cpp

```diff
@@ -160,11 +158,11 @@ ReturnStatuses KvaserCan::close()
   return ReturnStatuses::OK;
 }
 
-ReturnStatuses KvaserCan::read(int64_t *id,
-                               uint8_t *msg,
-                               uint32_t *size,
-                               bool *extended,
-                               uint64_t *time)
+ReturnStatuses KvaserCan::read(uint32_t * id,
```

Comment:
Space on both sides is specifically against the Google C++ style guide, and you've done it a bunch in this file. Is this intentional? https://google.github.io/styleguide/cppguide.html#Pointer_and_Reference_Expressions

Reply:
You're right. I assumed it was OK because it is the new style used in ROS2 but apparently they break from Google with ROS2. I'll fix them.